README.md

```diff
@@ -225,3 +225,82 @@ Example:
     ```
     pytest -s ecs/mxnet/training/test_ecs_mxnet_training.py::test_ecs_mxnet_training_dgl_cpu
     ```
+
+7. To run sagemaker Local tests, Launch a cpu or gpu instance based with latest deep learning AMI.
+    a) clone your github branch with changes and run the following commands
+       ```
+       git clone https://github.com/(github_account_id)/deep-learning-containers/
+       cd deep-learning-containers && git checkout (branch_name)
+       ```
+    b) login into the ecr repo where the new docker images build are present
+       ```
+       $(aws ecr get-login --no-include-email --registry-ids 669063966089 --region us-west-2)
+       ```
+    c) change to the sagemaker_tests/(framework)/(job_type) based the image to be tested
+       The example below refers to testing mxnet_training images
+       ```
+       cd test/sagemaker_tests/mxnet/traning/
+       pip3 install -r requirements.txt
+       ```
+    d) Run the sagemaker local tests using the below pytest command for images other than
```

Comment:
Suggested change:

"To run the SageMaker local mode tests (aside from tensorflow-inference), use the pytest command below:"